cli/pkg/local/app.go

```diff
@@ -434,6 +436,39 @@ func (a *App) infoHandler(info *localInfo) http.Handler {
 	})
 }
 
+// versionHandler servers the version struct.
+func (a *App) versionHandler(ctx context.Context) http.Handler {
+	// Get the latest version available
+	latestVersion, err := update.LatestVersion(ctx)
+	if err != nil {
+		a.Logger.Warnf("error finding latest version: %v", err)
+	}
+
+	inf := &versionInfo{
+		CurrentVersion: a.Version.Number,
+		LatestVersion:  latestVersion,
+	}
+
+	return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		data, err := json.Marshal(inf)
```

Comment:
1. Move this into the handler, so it doesn't block the initial server startup (it will be cached inside `update.LatestVersion` anyway)
2. Use `r.Context()` and don't accept `ctx` as parameter